cli/src/artifacts.ts

```diff
@@ -58,15 +62,15 @@ export default class Artifacts {
     if (!JSON.isObject(json)) {
       throw new TypeError("expected object, found " + (json === null ? "null" : typeof json));
```

Comment:
I've published a 0.3.1 update of `ts-typed-json` with customizable type error messages for type casts, so in places like this we could do:

```ts
json = JSON.asObject(json, 'json');
```